### PR TITLE
Release without prestable

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
- Do not create prestable release
- to be continued

Relates to #34900

### CODE REVIEW

**Comment by Felixoid** [comment]
I've tested it on Felixoid/ClickHouse and it looks good, but let's merge after tomorrow's release.

File: tests/ci/release.py [hunk 1]
```diff
@@ -170,26 +183,24 @@ def log_rollback(self):
...
         # Create release branch
         self.read_version()
         with self._create_branch(self.release_branch, self.release_commit):
             with self._checkout(self.release_branch, True):
                 self.read_version()
-                self.version.with_description(VersionType.PRESTABLE)
-                with self._create_gh_release(True):
-                    with self._bump_prestable_version():
+                self.version.with_description(self.get_stable_release_type())
+                with self._create_gh_release(False):
```
Comment (vdimir) [inline tests/ci/release.py:194]: Seems `self._create_gh_release(True)` is not called anymore, so can we remove argument?

**Review by vdimir — APPROVED** [review]
So we want to get rid of `prestable` and have `testing` and `stable`. Did I get it right?

File: tests/ci/release.py (hunk 1 above)
Comment (Felixoid) [reply tests/ci/release.py:194]: I'll leave it for our `-testing` releases. They will be pre-release releases

**Comment by Felixoid** [comment]
> So we want to get rid of `prestable` and have `testing` and `stable`. Did I get it right

Yes, testing and stable/lts

**Pull request merged**